content/de/privacy.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Datenschutzerklärung
+slug: privacy
+top_graphic: 4
+date: 2018-06-22
+lastmod: 2018-06-22
+---
+
+# Let's Encrypt Datenschutzerklärung
+
+{{< lastmod >}}
+
```

Comment:
The privacy policy (privacy.md) and the Non-Discrimination Statement and Policy (nondiscrimination.md) may be translated, but with a note on top saying that the English version is canonical.
https://github.com/letsencrypt/website/blob/master/TRANSLATION.md